script/crawl.sh

```diff
@@ -360,8 +360,8 @@ crawl "mongolian" \
   "https://github.com/bataak/dict-mn/archive/main.zip"
 # Hasn’t been updated in 10 years.
 crawl "nepali" \
-  "http://ltk.org.np" \
-  "http://ltk.org.np/downloads/ne_NP_dict.zip"
+  "https://ltk.org.np/nepali_spellchecker.php" \
```

Comment:
I don’t know whether this change in page is a good idea. This is more about the source website/project/people, rather than the actual page where it is downloaded from.

Reply:
I think I just assumed it was meant to be the page that contained the download link - I'll restore the previous URL.

Reply:
yes please, thanks!

Reply:
Updated now, so only the `http` -> `https` change is left.